hedera-node/src/main/java/com/hedera/services/store/contracts/precompile/HTSPrecompiledContract.java

```diff
@@ -187,6 +191,16 @@
 	protected static final int ABI_ID_DISSOCIATE_TOKENS = 0x78b63918;
 	//dissociateToken(address account, address token)
 	protected static final int ABI_ID_DISSOCIATE_TOKEN = 0x099794e8;
+	//createFungibleToken(HederaToken memory token, uint initialTotalSupply, uint decimals)
+	protected static final int ABI_ID_CREATE_FUNGIBLE_TOKEN = 0x7812a04b;
+	//createFungibleTokenWithCustomFees(HederaToken memory token, uint initialTotalSupply, uint decimals, FixedFee[] memory fixedFees, FractionalFee[] memory fractionalFees)
+	protected static final int ABI_ID_CREATE_FUNGIBLE_TOKEN_WITH_FEES = 0x4b37e995;
+	//createNonFungibleToken(HederaToken memory token)
+	protected static final int ABI_ID_CREATE_NON_FUNGIBLE_TOKEN = -0x6238ee20;
+	//createNonFungibleTokenWithCustomFees(HederaToken memory token, FixedFee[] memory fixedFees, FractionalFee[] memory fractionalFees, RoyaltyFee[] memory royaltyFees)
+	protected static final int ABI_ID_CREATE_NON_FUNGIBLE_TOKEN_WITH_FEES = -0x21abcac2;
+	//setTokenThresholdKeys (address token, TresholdTokenKey[] memory tokenKeys)
+	protected static final int ABI_ID_UPDATE_TOKEN_WITH_THRESHOLD_KEYS = 0x3ff927f;
```

Comment:
Since we decided to not implement Token Update now we should remove this function definition and only add support for create functions